build.gradle

```diff
@@ -18,18 +18,25 @@ buildscript {
 }
 
 plugins {
+    id "com.diffplug.gradle.spotless" version "3.23.1"
     id "java"
+    id "maven"
     id "org.owasp.dependencycheck" version "5.1.1"
-    id "com.diffplug.gradle.spotless" version "3.23.1"
     id "org.sonarqube" version "2.7.1"
 }
 
+repositories {
+    maven {
+        url "http://nexus.phx.connexta.com:8081/nexus/content/groups/public"
+    }
+}
```

Comment:
❓What's the difference between this `repositories` section and the one in `subprojects`? Can we move all of the repositories here?

Reply:
Consolidated repository blocks